src/glossary.md

```diff
@@ -7,15 +7,22 @@ the structure of the program when the compiler is compiling it.
 
 ### Arity
 
-Arity refers to the number of arguments a function or operation takes.
-For example, `(2, 3)` and `(4, 6)` have arity 2, and`(8, 2, 6)` has arity 3.
+Arity refers to the number of arguments a function or operator takes.
+For some examples, `f(2, 3)` and `g(4, 6)` have arity 2, while `h(8, 2, 6)`
+has arity 3. The `!` operator has arity 1.
 
 ### Array
 
 An array, sometimes also called a fixed-size array or an inline array, is a value
 describing a collection of elements, each selected by an index that can be computed
 at run time by the program. It occupies a contiguous region of memory.
 
+### Associated Item
+
+An associated item is an item that is associated with another item. Associated
```

Comment:
I'm okay with merging this, but this definition *is* a bit circular; we should figure out a better way to describe this.

Reply:
I'll do this as a part of Part 2.